### PR TITLE
chore(org): Set org id for the default internal org

### PR DESCRIPTION
**Pull request by dashed**
Before snowflake IDs was introduced, the internal Sentry org was assumed to have an ID of `1`. This assumption is relied upon for both `sentry` and `getsentry`, especially for granting superuser/sudo privileges. 

With snowflake IDs, the internal Sentry org will have an id that is an arbitrarily large positive integer.

Here, I'm changing so that we create the internal Sentry org with the id from `SENTRY_ORGANIZATION` option (which was introduced years ago). And I've updated the `SENTRY_ORGANIZATION` to have a value of `1` by default.

### CODE REVIEW

File: src/sentry/receivers/core.py [hunk 1]
```diff
@@ -69,7 +69,9 @@ def create_default_project(id, name, slug, verbosity=2, **kwargs):
     except IndexError:
         user = None
 
-    org, _ = Organization.objects.get_or_create(slug="sentry", defaults={"name": "Sentry"})
+    org, _ = Organization.objects.get_or_create(
+        slug="sentry", defaults={"name": "Sentry"}, id=getattr(settings, "SENTRY_ORGANIZATION", 1)
```
Comment (nhsiehgit) [inline src/sentry/receivers/core.py:73]: 🤔 

so this is defaulting the org id to 1? rather than enabling the snowflake id?

Comment (nhsiehgit) [reply src/sentry/receivers/core.py:73]: Should we instead look for a way to enable snowflake id's internally instead?

Comment (dashed) [reply src/sentry/receivers/core.py:73]: Yes. That's the idea.

Comment (dashed) [reply src/sentry/receivers/core.py:73]: @nhsiehgit it's already turned on by default here https://github.com/getsentry/sentry/blob/c31d98536a04f8eafd2952d568386321b7aa7b35/src/sentry/conf/server.py#L2824

**Comment by dashed** [comment]
We no longer need this as snowflake IDs are off by default in https://github.com/getsentry/sentry/pull/39714

**Pull request closed without merging**